app/views/va_common/_footer.html.erb

```diff
@@ -131,9 +125,16 @@ UserVoice.push(['addTrigger', {trigger_position: 'bottom-right', strings: {sugge
 UserVoice.push(['autoprompt', {}]);
 </script>
 
-<script>
-  $(document).foundation();
-</script>
+<!-- Google Tag Manager -->
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WFJWBD"
+height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-WFJWBD');</script>
+<!-- End Google Tag Manager -->
```

Comment:
The way this was set up (as documented [here](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/4eea43ed5d3be18de640b04bea76ac27605cfcd3/Products/analytics_procedure/readme.md#why-did-we-remove-the-tracking-code-from-the-gem)) each application currently includes its own Google Tag Manager code, rather than having a global version in this gem, so I believe this block should be removed.